### PR TITLE
Suggestions for sections i and ii - based on most recent develop branch

### PR DESCRIPTION
**Pull request by tlssuse**
Suggestions for i and ii sections on develop branch.

### CODE REVIEW

File: adoc/ii-Strategy/SA.adoc [hunk 1]
```diff
@@ -7,59 +7,44 @@ The strategy elements are typically used to model the strategic direction and ch
...
-Costs can be controlled as the cloud model is a ‘pay as you use’ scenario and many cloud vendors provide several consumption options to provide more choice to their customers. 
-+    
-This is an area where SUSE can help together with the cloud providers to offer the right options, but this also means you need to adapt to how you use the resources and the SAP software.
+Reduce Costs:: Many businesses experience large CapEx (Capital Expenditure) to maintain IT infrastructure, with purchases often in advance of actual need.  SUSE, together with the cloud providers, help businesses transform to a more efficient OpEx (Operating Expenditure) model, with ‘pay as you use’ options that empower businesses with greater agility.
  
-Scalability & Flexibility:: With the cloud, you can scale up your system as and when required, add or remove resources when the business demands it and as mentioned above, "rightsizing" and instance optimization to provide an efficient solution.
+Scalability & Flexibility:: With ready-to-run SUSE images in the cloud providers' marketplaces, businesses can scale up or scale down resources as required.  "Rightsizing" and instance optimization becomes much easier.
```
Comment (petersatsuse) [inline adoc/ii-Strategy/SA.adoc:33]: It now say something different - its not because of "ready-to-run" suse images, its because of the flexibility and possibilities what the cloud offers - so "Scalability & Flexibility" is because of the cloud infra and not because the sue images.
And btw. the images are not "ready to run"  - they are as small as possible (just enough os) and must be tailored to the needs/application/workload.

**Review by petersatsuse — CHANGES_REQUESTED** [review]
could you pls. have a look at then comments, and maybe refine the wording/meaning

File: adoc/i-Introduction/SA.adoc [hunk 2]
```diff
@@ -8,32 +8,25 @@ When SUSE first released SUSE Linux Enterprise Server for SAP Applications, it a
...
+*Simplify*::      the deployment of an SAP Landscape in the public cloud for development, test, and production.
 
 *Modernize*::     customer environments by taking advantage of the benefits of the public cloud.
 
-*Accelerate*::    customer migrations to the public cloud.
+*Accelerate*::    customer deployments and migrations to the public cloud.
 
 
-Building the infrastructure to run SAP Applications can be complex and demands significant effort, especially if a manual deployment method is used. When delivering multiple environments, for multiple engagements, reproducing the deployment can be tedious and error-prone. 
+Building the infrastructure to run SAP Applications can be complex and demand significant effort, especially if a manual deployment method is used. When delivering multiple environments, for multiple engagements, reproducing the deployment can be tedious and error-prone. Additional complexities arise from ensuring the infrastructure is highly available.
```
[the remark this answers is not in the document]
Comment (tlssuse) [reply adoc/i-Introduction/SA.adoc:18]: Understood.  It did not appear as a separate bullet in the PDF version, but I agree with your reason to put HA on a separate line.

File: adoc/ii-Strategy/SA.adoc (hunk 1 above)
Comment (tlssuse) [reply adoc/ii-Strategy/SA.adoc:33]: Understood.  Would the PAYG SUSE images facilitate this scalability and flexibility?

Comment (petersatsuse) [reply adoc/ii-Strategy/SA.adoc]: No - payg and byos - have the same packages - its only a different way of "selling" - technical they are 99.99 identical.

payg support the idea of cloud a bit better, as you simply can consume it and dont need to take care to buy and register a subscription from/with suse.

As said, the images have not much to do with scalability and flexibility (of the cloud)

Comment (tlssuse) [reply adoc/ii-Strategy/SA.adoc]: Yes, it is a bit of a stretch.  If a business wants to scale dynamically, though, using PAYG subscriptions might be easier than having to purchase enough subscriptions ahead of time.  I really just want to make sure here that we are able to highlight some advantage SUSE provides for scalability & flexibility.

File: adoc/ii-Strategy/SA.adoc [hunk 3]
```diff
@@ -7,59 +7,45 @@ The strategy elements are typically used to model the strategic direction and ch
...
 
-Reduce Costs:: Many businesses spent a lot of their capital on maintaining their IT Infrastructure. With help of the cloud, they can transform their CapEx (Capital Expenditure) to stable OpEx (Operating Expenditure).
-Costs can be controlled as the cloud model is a ‘pay as you use’ scenario and many cloud vendors provide several consumption options to provide more choice to their customers. 
-+    
-This is an area where SUSE can help together with the cloud providers to offer the right options, but this also means you need to adapt to how you use the resources and the SAP software.
+Reduce Costs:: Many businesses experience large CapEx (Capital Expenditure) to maintain IT infrastructure, with purchases often in advance of actual need.  SUSE, together with the cloud providers, help businesses transform to a more efficient OpEx (Operating Expenditure) model, with ‘pay as you use’ options that empower businesses with greater agility.
```
Comment (petersatsuse) [inline adoc/ii-Strategy/SA.adoc:33]: pls. add the other to bullet points again - its too simplified and does not show that you need to do something on the applicaton (SAP) side too - if you want to reduce costs!

File: adoc/ii-Strategy/SA.adoc [hunk 4]
```diff
@@ -7,59 +7,45 @@ The strategy elements are typically used to model the strategic direction and ch
...
-+
-SUSE's SAP HA Automation can add to these services to provide less downtime of your SAP application.
-    
-Remote access:: With the cloud, your employees can access data from anywhere, and at any time via the internet making business more flexible and increase the productivity of the employees.
-+    
-SUSE products natively provide many options for remote access and control.
-    
-So overall, public clouds offer significant benefits for all customers regardless of size. The use of public cloud resources can lower infrastructure costs and improve the scalability, agility, flexibility and availability of applications. 
+Resiliency:: Uptime is of prime importance to ensure day-to-day business operations run smoothly.  Moving to the cloud maximizes uptime and reduces downtime.
+The cloud improves disaster recovery and business continuity without the need to spend a huge amount of capital on robust disaster recovery tools.  Cloud providers offer a variety of services to help protect businesses from security threats and outages.  SUSE's SAP HA automation augments these services to further reduce downtime of SAP applications.
```
Comment (petersatsuse) [inline adoc/ii-Strategy/SA.adoc:40]: similar here - we always have two layers : one is the cloud with the benefits, second how suse can make it better,etc. 
If you move it into one sentence its less highlighting...
But this is maybe only may view....

File: adoc/ii-Strategy/SA.adoc [hunk 5]
```diff
@@ -7,59 +7,45 @@ The strategy elements are typically used to model the strategic direction and ch
...
+
+Remote access:: The cloud allows employees to access data from anywhere and at any time, making business more flexible and increasing productivity.  SUSE products natively provide many options for remote access and control.
```
Comment (petersatsuse) [inline adoc/ii-Strategy/SA.adoc:42]: similar as above

File: adoc/i-Introduction/SA.adoc [hunk 6]
```diff
@@ -8,32 +8,27 @@ When SUSE first released SUSE Linux Enterprise Server for SAP Applications, it a
...
+Building the infrastructure to run SAP Applications can be complex and demand significant effort, especially if a manual deployment method is used. When delivering multiple environments, for multiple engagements, reproducing the deployment can be tedious and error-prone.
 
-One additional challenge is ensuring the infrastructure is highly available, as this will add more complexity and steps to the deployment.
+Highly available infrastructure configurations raise additional complexities that further delay time to value.
 
-When deploying and managing a large number of systems in an SAP Landscape, there is often a secondary need, getting an overview of what is happening in the environment after the install is complete.
+When deploying and managing a large number of systems in an SAP Landscape, there is often a secondary need, getting an overview of what is happening in the environment after the installation is complete.
```
Comment (petersatsuse) [inline adoc/i-Introduction/SA.adoc:23]: why did you remove the monitoring part?

**Review by petersatsuse — COMMENTED** [review]
pls have a look

File: adoc/ii-Strategy/SA.adoc (hunk 3 above)
Comment (tlssuse) [reply adoc/ii-Strategy/SA.adoc:33]: I'll merge in an update to this effect.
